src/app/collective-rewards/CollectiveRewardsPage.tsx

```diff
@@ -30,12 +32,12 @@ export const CollectiveRewardsPage = () => {
           className="flex flex-col gap-10 px-4 py-8 md:px-6 md:pt-6 md:pb-10 bg-v3-bg-accent-80"
           title={
             <Header variant="h3" caps>
-              THE REWARDS AT WORK - CURRENT CYCLE
+              THE REWARDS AT WORK - {!isDesktop && <br />} CURRENT CYCLE
```

Comment:
Im not sure about this, maybe we can check this tomorrow during the parking lot.